src/common/token.ts

```diff
@@ -190,3 +203,145 @@ export class ArweaveToken implements TokenTools {
 export const WinstonToTokenAmount = (winston: BigNumber.Value) => winston;
 export const ARToTokenAmount = (ar: BigNumber.Value) =>
   new BigNumber(ar).times(1e12).valueOf();
+
+export class SolanaToken implements TokenTools {
```

Comment:
nit: put these in separate files `tokens/solana.ts`

Reply:
3b9827f